src/lib/profile.js

```diff
@@ -12,7 +12,20 @@ function Profile(items) {
     }
   }
 
+  function getAssumedRole(elId) {
+    var el = document.getElementById(elId);
+    if (!el) return null;
+
+    var role = el.textContent;
+    if (role.indexOf('/') > -1) {
+      return role.split("/")[0];
+    } else {
+      return role;
+    }
```

Comment:
The only `role.split("/")[0];` is enough, `if ~` is not needed.

Reply:
Refactored.